setup-Hhh/tt/unc-Hhh-8TeV-00.conf

```diff
@@ -3,17 +3,15 @@ lumi_8TeV                               	lnN
 CMS_eff_t_tautau_8TeV                           lnN  
 # CMS_eff_t_Hhh_tautau_8TeV                  shape
 CMS_scale_t_tautau_8TeV                     	shape 
-CMS_scale_j_8TeV                            	lnN   
+CMS_scale_j_8TeV                            	shape   
 CMS_eff_b_8TeV		        		lnN
 CMS_fake_b_8TeV		        		lnN
-CMS_htt_WNorm_tautau_2jet0tag_8TeV         	lnN   
 CMS_htt_QCDSyst_tautau_2jet0tag_8TeV       	lnN   
 CMS_htt_zttNorm_8TeV                    	lnN   
-CMS_htt_extrap_ztt_tautau_2jet0tag_8TeV           lnN   
-CMS_htt_ttbarNorm_8TeV                  	lnN   
-CMS_htt_ttbarNorm_tautau_2jet0tag_8TeV         	lnN   
+CMS_htt_extrap_ztt_tautau_2jet0tag_8TeV         lnN   
 CMS_htt_DiBosonNorm_8TeV                	lnN
-CMS_htt_DiBosonNorm_tautau_2jet0tag_8TeV       	lnN
+CMS_htt_ttbarNorm_8TeV                  	lnN   
+CMS_htt_ttbar_emb_8TeV 		        	lnN   
```

Comment:
CMS_htt_ttbar_emb_8TeV is included here but doesn't appear in the .vals file for this category. Please check if it should be included or not.